src/node_errors.h

```diff
@@ -36,10 +36,12 @@ void OnFatalError(const char* location, const char* message);
   V(ERR_CRYPTO_TIMING_SAFE_EQUAL_LENGTH, RangeError)                           \
   V(ERR_CRYPTO_UNKNOWN_CIPHER, Error)                                          \
   V(ERR_CRYPTO_UNKNOWN_DH_GROUP, Error)                                        \
+  V(ERR_DLOPEN_FAILED, Error)                                                  \
```

Comment:
Unless I am missing something...these codes are not documented yet?

Reply:
Good catch. At some point should likely include node_errors.h in the lint rule that checks those.